src/main/java/org/gitlab4j/api/GitLabApi.java

```diff
@@ -1674,6 +1675,21 @@ public WikisApi getWikisApi() {
         return wikisApi;
     }
 
+    /**
+     * Gets the KeysApi instance owned by this GitLabApi instance. The KeysApi is used to look up users by their ssh key signatures
+     *
+     * @return the KeysApi instance owned by this GitLabApi instance
+     */
+    public KeysApi getKeysAPI() {
```

Comment:
Can you split KeysApi implementation and MembershipSourceType fix in 2 differents PR?

Reply:
Done: https://github.com/gitlab4j/gitlab4j-api/pull/814